Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "RegistryCI"
 uuid = "0c95cc5f-2f7e-43fe-82dd-79dbcba86b32"
 authors = ["Dilum Aluthge <dilum@aluthge.com>", "Fredrik Ekre <ekrefredrik@gmail.com>", "contributors"]
-version = "6.2.0"
+version = "6.3.0-DEV"
```

Comment:
I generally prefer to release immediately after merging a PR. I.e. I only bump to a `-DEV` version if I'm planning on making multiple PRs in a row in a short period of time.